### ISSUE TITLE
Null-coalescing operator (??) flagged as invalid

### ISSUE THREAD

**Issue by thijskaspers** [asm89/twig-lint#42]
Using `??` in a Twig templated is flagged as invalid with the following warning:

````
Unexpected token "punctuation" of value "?"
````
It is a default Twig operator (https://twig.symfony.com/doc/2.x/templates.html#other-operators) so it should be valid?

**Comment by asm89** [comment]
I'm guessing we need support for more newish operators.

**Comment by Snowbaha** [comment]
Hello, are you planning to add this feature? If not, do you know how to exclude this false warning?
